.gitlab-ci.yml

```diff
@@ -214,38 +213,44 @@ trigger_internal_operator_image:
   stage: release
   rules:
     - if: $CI_COMMIT_TAG
-      variables:
-        IMAGE_RELEASE_PROD: "true"
-        IMAGE_TAG: $CI_COMMIT_REF_SLUG
-    - if: $PUSH_IMAGES_TO_STAGING == 'true'
-      when: manual
-      variables:
-        IMAGE_TAG: $CI_COMMIT_REF_SLUG-$CI_COMMIT_SHORT_SHA
     - when: never
   trigger:
     project: DataDog/images
     branch: master
     strategy: depend
   variables:
     IMAGE_VERSION: tmpl-v1
-    IMAGE_NAME: datadog-operator
-    RELEASE_TAG: $IMAGE_TAG
-    BUILD_TAG: $IMAGE_TAG
+    IMAGE_NAME: $PROJECTNAME_CHECK
     TMPL_SRC_IMAGE: v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
+    RELEASE_TAG: ${CI_COMMIT_REF_SLUG}
+    BUILD_TAG: ${CI_COMMIT_REF_SLUG}
     RELEASE_STAGING: "true"
-    RELEASE_PROD: $IMAGE_RELEASE_PROD
+    RELEASE_PROD: "true"
 
 trigger_internal_operator_check_image:
   stage: release
   rules:
     - if: $CI_COMMIT_TAG
-      variables:
-        IMAGE_RELEASE_PROD: "true"
-        IMAGE_TAG: $CI_COMMIT_REF_SLUG
+    - when: never
+  trigger:
+    project: DataDog/images
+    branch: master
+    strategy: depend
+  variables:
+    IMAGE_VERSION: tmpl-v1
+    IMAGE_NAME: $PROJECTNAME_CHECK
+    TMPL_SRC_IMAGE: v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
+    RELEASE_TAG: ${CI_COMMIT_REF_SLUG}
+    BUILD_TAG: ${CI_COMMIT_REF_SLUG}
+    RELEASE_STAGING: "true"
+    RELEASE_PROD: "true"
+
+
+trigger_internal_staging_only_operator_image:
```

Comment:
could we call this something like `trigger_custom_operator_image_staging` (and similarly for the operator check)? to distinguish its use more from the above job `trigger_internal_operator_image`